oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/FormFeedback/FormFeedback.jsx

```diff
@@ -26,11 +26,22 @@ const CustomMessage = ({ children, ...uiProps }) => {
     setValues(valuesWithoutInternalErrorFields);
   };
   return (
-    <Message onDismiss={handleDismiss} className="rel-mb-2" {...uiProps}>
+    <Message
+      onDismiss={handleDismiss}
+      className="rel-mb-2 form-feedback"
+      {...uiProps}
+    >
       {children}
     </Message>
   );
 };
+
+const scrollToField = (fieldName) => {
```

Comment:
Should be extracted to utility functions (generalize by accepting just the selector argument, use here like `scrollTo(\`label[for=${fieldName}]\`)`